### ISSUE TITLE
checkra1n app not installing since version 0.10.1

### ISSUE THREAD

**Issue by kartana** [foxlet/bootra1n#42]
It worked before and I tried multiple times now, no checkra1n app. It jailbreaks fine though.

iPhone SE on 13.3. I am trying to restore rootFS to upgrade iOS.

**Comment by AnnonZerp** [comment]
Remove your pin code

**Comment by snakeskin592** [comment]
i think the same think has happened to me, because i cant seem to get it to work anymore. USBMUX errror 78 and it keeps occuring even after i changed and swapped usb cables.

**Comment by kartana** [comment]
> Remove your pin code

Didn't change the result.

**Comment by akirakusui** [comment]
You just have to restart the PC and relaunch bootra1n. It may take many attempts but it will work. Make sure you have your passcode setting off.

**Comment by kartana** [comment]
> You just have to restart the PC and relaunch bootra1n. It may take many attempts but it will work. Make sure you have your passcode setting off.

NO, it does NOT install the Checkra1n app on my SE with 13.3! It does jailbreak it but doesn't install the app. I tried it countless times. I went back to Version 0.9.8 and this did install the app. So there is definitely something wrong with this version. I was able to restore rootfs and will now update to 13.4.1 and rejailbreak with 0.10.1 and see if that installs the app.
